prometheus/openshift/README.md

```diff
@@ -0,0 +1,37 @@
+## Prerequisites
+
+The following prerequisites must be met prior to beginning to deploy GitLab CE
+
+* 1 [Persistent Volume](https://docs.openshift.com/container-platform/latest/architecture/additional_concepts/storage.html) or a cluster that supports [dynamic provisioning with a default StorageClass](https://docs.openshift.com/container-platform/latest/install_config/storage_examples/storage_classes_dynamic_provisioning.html)
+* OpenShift Command Line Tool
+* [Openshift Applier](https://github.com/redhat-cop/openshift-applier/) to deploy GitLab CE. As a result you'll need to have [ansible installed](http://docs.ansible.com/ansible/latest/intro_installation.html)
+* The following traffic must be enabled for the Cluster Nodes on the underlaying Firewall
+    * 9100/TCP
+    * 1936/TCP
+    * 8443/TCP
+
+
+### Environment Setup
+
+1. Clone this repository: `git clone https://github.com/redhat-cop/monitoring.git`
+2. `cd monitoring/prometheus/openshift`
+3. Run `ansible-galaxy install -r requirements.yml --roles-path=galaxy`
+4. Login to OpenShift: `oc login -u <username> https://master.example.com:8443`
```

Comment:
I'd recommend making this a bit more generic - i.e.: `oc login -u <username> <openshift_fqdn>[:port(optional)]`